### DISCUSSION TITLE
失效了。。。

### DISCUSSION THREAD

**Discussion by zhangshuai-phys** [huiyadanli/RevokeMsgPatcher#656] [Q&A] [answered]
大佬，刚发现1.7对3.9.8.15失效了，尝试卸载了新版本微信，安装了旧版本，依然不能撤回信息，记得以前可以随时撤回已发消息的。

**Comment by huiyadanli** [comment, marked as answer]
看上去没撤回，实际已经撤回 #646

3.9.8.15我测试1.7是正常的
